src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -122,97 +123,117 @@ const WorkflowDashboard = _.flow(
+              style: { display: 'flex' },
+              tooltip: 'Job Manager'
+            }, [icon('tasks', { size: 18 }), ' Job Manager']),
+            h(Link, {
+              ...Utils.newTabLinkProps,
+              href: bucketBrowserUrl(`${bucketName}/${submissionId}/${workflowName}/${workflowId}`),
+              style: { display: 'flex', marginLeft: '1rem' },
+              tooltip: 'Execution directory'
+            }, [icon('folder-open', { size: 18 }), ' Execution Directory']),
+            h(Link, {
+              onClick: () => setShowLog(true),
+              style: { display: 'flex', marginLeft: '1rem' }
+            }, [icon('fileAlt', { size: 18 }), ' View execution log'])
+          ])
+        ])
+      ]),
+      failures && h(Collapse,
+        {
+          style: { marginBottom: '1rem' },
+          initialOpenState: true,
+          title: div({ style: Style.elements.sectionHeader }, [
+            'Workflow-Level Failures',
+            h(ClipboardButton, {
+              text: JSON.stringify(failures, null, 2),
+              style: { marginLeft: '0.5rem' },
+              onClick: e => e.stopPropagation() // this stops the collapse when copying
+            })
+          ])
+        }, [h(ReactJson, {
+          style: { whiteSpace: 'pre-wrap' },
+          name: false,
+          collapsed: 4,
+          enableClipboard: false,
+          displayDataTypes: false,
+          displayObjectSize: false,
+          src: restructureFailures(failures)
+        })]
+      ),
+      h(Collapse,
+        {
+          title: div({ style: Style.elements.sectionHeader }, ['Calls']),
+          initialOpenState: true
+        }, [
+          div({ style: { marginLeft: '1rem' } },
+            [makeSection('Total Call Status Counts', [
+              !_.isEmpty(calls) ? statusCell(workflow) : div({ style: { marginTop: '0.5rem' } }, ['No calls have been started by this workflow.'])
+            ]),
+            !_.isEmpty(calls) && makeSection('Call Lists', [
+              _.map(callName => {
+                return h(Collapse, {
+                  key: callName,
+                  style: { marginLeft: '1rem', marginTop: '0.5rem' },
+                  title: div({ style: { ...Style.codeFont, ...Style.elements.sectionHeader } }, [`${callName} × ${calls[callName].length}`]),
+                  initialOpenState: !_.every({ executionStatus: 'Done' }, calls[callName])
+                }, [
+                  h(CallTable, { namespace, name, submissionId, workflowId, callName, callObjects: calls[callName] })
+                ])
+              }, callNames)
+            ])]
+          )
+        ]
+      ),
+      wdl && h(Collapse,
+        {
+          title: div({ style: Style.elements.sectionHeader }, ['Submitted workflow script'])
+        }, [h(WDLViewer, { wdl })]
+      ),
+      showLog && h(UriViewer, { googleProject: namespace, uri: workflowLog, onDismiss: () => setShowLog(false) })
+    ])
+    )
```

Comment:
If close brackets of any kind have the same indentation, on two lines in a row, it means they were opened on the same line, so they should be closed on the same line for consistency.